src/main/java/calculator/CalculateController.java

```diff
@@ -0,0 +1,22 @@
+package calculator;
+
+public class CalculateController {
+
+    private final Calculator calculator;
+    private final Parser parser;
+
+    public CalculateController() {
+        this.calculator = new Calculator();
+        this.parser = new Parser();
+    }
+
+    public int calculate (String input){
+
+        if (input == null || input.isEmpty()) {
+            return 0;
+        }
+
+        String [] tokens = parser.split(input);
```

Comment:
```suggestion
        String[] tokens = parser.split(input);
```
array선언시 공백을 추가해주시는것이 어떤 의미를 가지는지 알수 있을까요 ?
개인적으로는 위와같이 선언하곤 합니다.

Reply:
따로 의미는 없습니다! 리뷰해주신대로 수정하겠습니다.